src/Symfony/Component/Messenger/Stamp/TransportConfigurationStamp.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+class TransportConfigurationStamp implements StampInterface
+{
+    private $delay = 0;
+
+    // TODO - add more stuff!
```

Comment:
And/or move away from type safety and add a generic “data” store

Reply:
Indeed. I think there are a few things that we should generically support - like delay, priority, maybe ttl. From a DX perspective, I'd like these to have proper setter methods. But then there will also need to be dynamic, transport-specific stuff, like `routingKey`, which would require a generic data store. So... both?

Reply:
Say getters and setters for “normal” stuff. But we should only use one property. Then one could easily implement “getAll()”

That would be super dynamic for MyCustomTransport.

Reply:
I'd vote in favour of more specific stamps actually instead of a generic one. `DelayedStamp`, `PriorityStamp`, etc... it doesn't cost much and makes everything much more explicit.

Reply:
Yes, I like this - it's even cleaner/easier when you're creating the envelope and setting this stuff on the stamps. I've just added `DelayStamp`